borsh/tests/test_simple_structs.rs

```diff
@@ -171,6 +171,8 @@ fn test_simple_struct() {
     let e = E { a: &a };
     let encoded_ref_a = e.try_to_vec().unwrap();
     assert_eq!(encoded_ref_a, encoded_a);
+    #[cfg(feature = "std")]
+    insta::assert_debug_snapshot!(encoded_a);
```

Comment:
This is a pity that we skip the format safety checks for no-std. Is there anything we can do about it? 🤔

Reply:
nope, not with `insta` crate . it explicitly imports `std` in expanded macros

Reply:
https://crates.io/crates/expect-test and https://crates.io/crates/k9 and don't feature `no_std` as far as i can see as well

Reply:
Ok, let's live with this for now. Maybe someone will contribute no_std support to insta or some other similar tool.